docs/src/pages/css-in-js/basics/basics.md

```diff
@@ -19,7 +19,7 @@ We think that this is the future:
 Material-UI's styling solution is inspired by many other CSS-in-JS libraries like [styled-components](https://www.styled-components.com/) and [emotion](https://emotion.sh/).
 
 - 💅 You can expect [the same advantages](https://www.styled-components.com/docs/basics#motivation) as styled-components.
-- 🚀 Is [blazing fast](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-benchmark/README.md#material-uistyles).
+- 🚀 It's [blazing fast](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-benchmark/README.md#material-uistyles).
```

Comment:
"blazingly fast"... But according to google "blazing fast" is more frequently used, so let's go with that. (Who needs grammar anyway! 😄 )